app/views/layouts/_modal_script.html.erb

```diff
@@ -0,0 +1,217 @@
+<script src="https://unpkg.com/micromodal/dist/micromodal.min.js"></script>
+<script>
```

Comment:
Should this even be an ERB file? Might be better to just have this be a .js file that then gets loaded via <script> tags in the layout directly?

I ask because there's a bunch of syntax irregularities in this file that don't get picked up by `prettier`.

Reply:
No, it was just taking after tracking scripts, but I guess that file was purposely trying to avoid being picked up by prettier.